src/main/java/codes/biscuit/skyblockaddons/mixins/MixinEntityRenderer.java

```diff
@@ -49,7 +49,7 @@ private void removeEntities(List<Entity> list) {
             if (!SkyblockAddons.getInstance().getConfigValues().isDisabled(Feature.DISABLE_PROFILE_VIEW_IF_DRAGON_ALIVE)) {
                 Minecraft mc = Minecraft.getMinecraft();
                 for (Entity elist : mc.theWorld.loadedEntityList) {
-                    if (elist instanceof EntityEnderman) {
+                    if (elist instanceof EntityDragon) {
                         double coreX = -670;
```

Comment:
Shouldn't this be final?

Reply:
I was testing with Endermans (Zealots) and forgot to change it back to Ender Dragon, the doubles should be fine though.